packages/react/src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+interface TabPanelProps {
+  index: number;
+  value: number;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+const TabPanel = (props: TabPanelProps) => {
+  const { children, index, value, className, ...other } = props;
+
+  return (
+    <div
+      hidden={value !== index}
```

Comment:
`hidden` isn't supported universally, so may be safer to conditionally display only the currently visible panel or just ensure that's visibly hidden via css.